src/ploneintranet/api/tests/test_microblog.py

```diff
@@ -0,0 +1,25 @@
+from ploneintranet.api.testing import IntegrationTestCase
+from ploneintranet import api as pi_api
+from ploneintranet.microblog.statusupdate import StatusUpdate
+
+
+class TestMicroblog(IntegrationTestCase):
+
+    def test_create_statusupdate(self):
+        text = 'Hello this is my update'
+        update = pi_api.microblog.create_statusupdate(text)
+        self.assertIsInstance(update, StatusUpdate)
+        self.assertEqual(update.text, text)
+
```

Comment:
Shouldn't it rather be something like `api.microblog.statusupdate.get(text)` ? 
or `api.statusupdate.get(text)`? 

That seems more in line with the way plone.api does it.